src/views/pages/rooms/components/RoomList/styled.ts

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { TitleH2 } from 'views/components/Title';
+import { Button } from 'views/components/Button';
+import { mobileDevice } from 'views/theme/mediaQuery';
+
+export const RoomBaseLayout = styled.div`
+  background-color: ${({ theme }) => theme.colors.backgroundSecondary};
+  height: 60vh;
```

Comment:
do not set height, it should adoption to any place you put it in. so height is managed by tab content in rooms or it === content height